synapse/lib/snap.py

```diff
@@ -88,6 +88,7 @@ def getStormRuntime(self, opts=None, user=None):
             user = self.user
 
         runt = s_storm.Runtime(self, opts=opts, user=user)
+        runt.isModuleRunt = True
```

Comment:
so every runtime made directly from the snap itself is a modulerunt. is there ever a case where we wouldn't want that?

Reply:
Hmmm. If there's no function calls or definitions, isModuleRunt has no effect. If there's only function definitions, those will get recorded but nothing beyond that. So outside of functions, this won't really impact anything. And since runts are specfic to a particular query instance, there's no pollution of isModuleRunt across runtimes.

TL;DR: I can't think of one.